.scripts/run_osx_build.sh

```diff
@@ -9,7 +9,7 @@ MINIFORGE_HOME=${MINIFORGE_HOME:-${HOME}/miniforge3}
 ( startgroup "Installing a fresh version of Miniforge" ) 2> /dev/null
 
 MINIFORGE_URL="https://github.com/conda-forge/miniforge/releases/latest/download"
-MINIFORGE_FILE="Miniforge3-MacOSX-$(uname -m).sh"
+MINIFORGE_FILE="Miniforge3-MacOSX-x86_64.sh"
```

Comment:
@hmaarrfk Can you revert the changes in this file? This breaks the local builds for me on `osx-arm64`?

Reply:
Would it be appropriate to open an issue with conda-smithy to make this the default if we specify something in `conda-forge.yml`?